src/features/landing/components/ForWhoSection/ForWhoSection.tsx

```
@@ -36,95 +36,64 @@ function ForWhoSection(): React.ReactElement {
+              alt={circle.alt}
+              aria-hidden="true"
+              sx={styles.circle}
+              loading="lazy"
+            />
           </Box>
         </Box>
-      </Container>
-      <Box sx={styles.smCardsWrapper}>
-        <Cards />
+        <Box sx={styles.square}>
+          <Box component="img" {...bigScreenProps} sx={styles.bigScreen} loading="lazy" />
```

ESLint

[error] Prop spreading is forbidden

(react/jsx-props-no-spreading)